### ISSUE TITLE
Row grouping not working

### ISSUE THREAD

**Issue by theiliad** [glideapps/glide-data-grid#958]
I've noticed that a lot of modules related to row-grouping don't see to be included in 6.0.3 release, not sure if previous releases did in fact include those

For instance these files

<img width="1356" alt="image" src="https://github.com/glideapps/glide-data-grid/assets/14989804/13ad4ca8-ad19-4962-a391-119efac33efb">

and

<img width="1210" alt="image" src="https://github.com/glideapps/glide-data-grid/assets/14989804/3e0156d8-4503-4bd8-ab53-667eabb750ab">

are missing from final build
https://unpkg.com/browse/@glideapps/glide-data-grid@6.0.3/src/data-editor/

I'm also seeing type warnings suggesting that this feature might not actually be in the package
![image](https://github.com/glideapps/glide-data-grid/assets/14989804/2557dbac-9650-42db-98c8-6245a93a2cda)

Also I'm not able to see any results after pulling all those modules into my local codebase, and adding rowGrouping to my grid

Is this in fact an issue?

**Comment by citizensas** [comment]
If it helps, it is available in the latest alpha `6.0.4-alpha8` release.

**Comment by theiliad** [comment]
> If it helps, it is available in the latest alpha `6.0.4-alpha8` release.

I see, so not stable yet?

**Comment by snowy-owll** [comment]
> I see, so not stable yet?

I couldn't get the row grouping to work correctly. It looks like there are problems with calculating the original row indexes in groups.

**Comment by mirus-ua** [comment]
> > I see, so not stable yet?
> 
> I couldn't get the row grouping to work correctly. It looks like there are problems with calculating the original row indexes in groups.

We are playing a bit with the grid for our POC, and here is a potential solution for your problem if it's the same as ours 
`const item = data[isGroupHeader ? originalIndex[1] : row];`

**Comment by snowy-owll** [comment]
> We are playing a bit with the grid for our POC, and here is a potential solution for your problem if it's the same as ours `const item = data[isGroupHeader ? originalIndex[1] : row];`

@mirus-ua Yes, I know about this. It helps with collapsing rows, but it returns a little wrong index. Even their example doesn't show all the data. It should display 1000 rows, but the last row index is 995. So, every group "eats" one row.

**Comment by lukasmasuch** [comment]
Closing this since its included in the alpha release and we also recently merged some fixes: https://github.com/glideapps/glide-data-grid/pull/999 and https://github.com/glideapps/glide-data-grid/pull/924
